tslearn/metrics.py

```diff
@@ -671,17 +628,30 @@ def compute_mask(s1, s2, global_constraint=0,
     """
     sz1 = s1.shape[0]
     sz2 = s2.shape[0]
-    if global_constraint == 2:
+    if (global_constraint == 0 and sakoe_chiba_radius is not None
+            and itakura_max_slope is not None):
+        raise ValueError("global_constraint is not set for DTW, but both"
```

Comment:
This is an option. An alternative would be to raise a warning and make an arbitrary choice (which wouldn't crash the code).

Reply:
OK, and then I guess the "ideal" default would be "no constraint", no?

Reply:
That is indeed the best option in my opinion as well.

Reply:
Done (locally, will push soon).